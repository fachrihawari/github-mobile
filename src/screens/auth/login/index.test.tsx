import "react-native";
import React from "react";
import render from "react-test-renderer";

import Login from "./index";

it("render default Login", () => {
  const tree = render.create(<Login />).toJSON();
  expect(tree).toMatchSnapshot();
});
