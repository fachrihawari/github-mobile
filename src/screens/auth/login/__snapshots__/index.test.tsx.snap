// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render default Login 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <TextInput
    allowFontScaling={true}
    autoCapitalize="none"
    autoFocus={true}
    onChangeText={[Function]}
    onSubmitEditing={[Function]}
    placeholder="Username"
    rejectResponderTermination={true}
    returnKeyType="done"
    style={
      Object {
        "backgroundColor": "#f8f8f8",
        "borderColor": "#ededed",
        "borderRadius": 22.5,
        "borderStyle": "solid",
        "borderWidth": 1,
        "fontSize": 18,
        "height": 45,
        "marginBottom": 15,
        "shadowColor": "rgba(191, 191, 191, 0.5)",
        "shadowOffset": Object {
          "height": 2,
          "width": 0,
        },
        "shadowOpacity": 1,
        "shadowRadius": 4,
        "textAlign": "center",
        "width": 600,
      }
    }
    underlineColorAndroid="transparent"
  />
  <View
    accessible={true}
    clickable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#0087E9",
            "borderRadius": 8,
            "justifyContent": "center",
            "paddingHorizontal": 20,
            "paddingVertical": 10,
          },
          Object {
            "borderRadius": 22.5,
            "height": 45,
            "width": 600,
          },
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "white",
              "fontSize": 16,
            },
            undefined,
          ]
        }
      >
        Continue
      </Text>
    </View>
  </View>
</View>
`;
